packages/vm/src/cache.rs

```diff
@@ -487,7 +487,7 @@ mod test {
         let env1 = mock_env(&instance1.api, "owner1", &coins(1000, "earth"));
         let msg1 = r#"{"verifier": "sue", "beneficiary": "mary"}"#.as_bytes();
         match call_init::<_, _, _, Never>(&mut instance1, &env1, msg1) {
-            Err(VmError::RuntimeErr { .. }) => (), // all good, continue
+            Err(VmError::GasDepletion { .. }) => (), // all good, continue
```

Comment:
I like this change of specific error. For the tests, I have stated using `unwrap_err()` to simplify such cases.

```rust
match call_init::<_, _, _, Never>(&mut instance1, &env1, msg1).unwrap_err() {
    VmError::GasDepletion { .. } => (), // all good, continue
    e => panic!("unexpected error, {:?}", e),
}
```

Reply:
ah cool, i'll do that here as well, i suppose.